docs/getting-started/docker.mdx

```diff
@@ -33,9 +33,14 @@ docker run -d \
   --name jellyseerr \
   -e LOG_LEVEL=debug \
   -e TZ=Asia/Tashkent \
-  -e PORT=5055 `#optional` \
+  -e PORT=5055 \
```

Comment:
Why not mark it as optional anymore?

Reply:
If you copy and paste the command, you get the error `#optional: command not found`. Therefore, I removed it to make it more user-friendly for beginners when they copy and paste.

Reply:
Makes sense. But could you mark it below if people want to change the internal port of jellyseerr?